test/basic_types/string.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const buildType = process.config.target_defaults.default_configuration;
+const assert = require('assert');
+
+test(require(`../build/${buildType}/binding.node`));
+test(require(`../build/${buildType}/binding_noexcept.node`));
+
+function test(binding) {
+	const testString = binding.basic_types_string.createUtf8String("κόσμε");
+	assert.strictEqual(testString, "κόσμε");
+}
```

Comment:
Assuming this is still a work in progress as its not calling createUtf16String yet, but good start.   Let us know when you think its complete and ready to review.

Reply:
Thank you for review ~
Yes, I am still a work in progress +_+//

I am considering how to make test for String type.
I need to understand String for ECMA script more.

Reply:
I think we can resolve this now, right? There is a call to `createUtf16String` now. *That* is what you were referring to, right, @mhdawson?